### DISCUSSION TITLE
当前版本的daijin 运行后选择任何选项后按回车就直接退出

### DISCUSSION THREAD

**Discussion by diggerdu** [RuriOSS/daijin#50] [General]
当前版本的daijin 运行后选择任何选项后按回车 tui就直接退出 请问这个bug怎么解决

另外 由于我编译的时候提示
unsupported option '-mshstk' for target 'aarch64-unknown-linux-android24'
因此我把ruri makefile里的shadow stack改成了

```
#Shadow Stack.
#SHADOW_STACK = -mshstk
SHADOW_STACK =
```

不知道是不是这个原因导致我无法执行

**Comment by diggerdu** [comment]
没事了 才发现daijin当前只实现了显示main menu

**Comment by Moe-hacker** [comment]
> 没事了 才发现daijin当前只实现了显示main menu

前端一点没干![frontenders-and-backenders.jpeg](https://github.com/Moe-hacker/daijin/assets/76621454/724b1214-1851-4eb9-8ac6-251ec8e18045)

**Comment by Moe-hacker** [comment]
我觉得它现在应该能用了![(2)image.jpeg](https://github.com/Moe-hacker/daijin/assets/76621454/ee860033-9774-4405-887b-ed4736ad3cb5)
